### ISSUE TITLE
Remove object shape tool

### ISSUE THREAD

**Issue by celeitner** [nhnent/tui.component.image-editor#16]
Shape tool is no longer available after removing the last shape.

1. click shape
2. create square
3. select square
4. click remove selected object
5. no longer can you create a shape

This also happens if I select the shape object and select delete on the keyboard. ( I suspect they both take the same code path - to removeActiveObject() )
I'm not sure where to look to debug this?

**Comment by dongsik-yoo** [comment]
The `removeActiveObject()`invokes `endDrawingShapeMode()`, therefore you should use `startDrawingShapeMode()` again after` removeActiveObject()` if you want to stay drawing shape mode.

Thanks a lot.

**Comment by celeitner** [comment]
That is what I am doing but it does not work. If you have only one shape and remove it then the startDrawingShapeMode() does not work anymore. You can see this in your demo.

click shape
create a square
select the square
click remove active object
no longer can you create a shape

**Comment by dongsik-yoo** [comment]
I could reproduce what you said. OK I can handle that problem in demo page.

**Comment by dongsik-yoo** [comment]
This problem was solved. The patch will be included in the next version.

**Comment by celeitner** [comment]
Can you share the details of the fix? When is the next version going to be released?

**Comment by dongsik-yoo** [comment]
Below code `this.deactivateAll();` line will be removed in the next release version. 

[imageEditor.js]
```
   /**
     * End current action & Deactivate
     * @example
     * imageEditor.startFreeDrawing();
     * imageEidtor.endAll(); // === imageEidtor.endFreeDrawing();
     *
     * imageEditor.startCropping();
     * imageEditor.endAll(); // === imageEidtor.endCropping();
     */
    endAll() {
        this.endTextMode();
        this.endFreeDrawing();
        this.endLineDrawing();
        this.endCropping();
        this.endDrawingShapeMode();
        this.deactivateAll();
        this._state = states.NORMAL;
    }
```
And we are planning to release the next version at the end of this month.

**Comment by dongsik-yoo** [comment]
This issue has been solved at [v2.0.0](https://github.com/nhnent/tui.component.image-editor/releases/tag/2.0.0).
Please check this out at [demo page](http://nhnent.github.io/tui.component.image-editor/latest/tutorial-basic.html).
